src/officehours_api/models.py

```diff
@@ -201,8 +201,16 @@ def start(self):
         if not backend:
             raise DisabledBackendException(self.backend_type)
         try:
+            attendee = self.attendees.first()
```

Comment:
can we concatenate names from multiple attendees?

Reply:
yes